src/pep8ext_naming.py

```diff
@@ -299,6 +318,9 @@ def visit_classdef(self, node, parents, ignore=None):
         name = name.strip('_')
         if not name[:1].isupper() or '_' in name:
             yield self.err(node, 'N801', name=name)
+        superclasses = self.superclass_names(name, parents)
+        if "Exception" in superclasses and name[-5:] != "Error":
```

Comment:
Is `name[-5:]` must faster than `name.endswith("Error")`? I think the latter is more readable, but performance could trump that here.

Reply:
I not encountered a high degradation of the  performance. So I take the 'endswith' suggestion. If performance issues are raised I'll rollback this change.